docs/service-brokers/docs/011-configuration-helm-broker.md

```diff
@@ -31,14 +31,15 @@ Follow these steps to change the configuration and make the Helm Broker fetch bu
             version: 0.0.1
       ```
 
-    - A `{bundle_name}-{bundle_version}.tgz` file for each bundle version defined in the `index.yaml` file. The `.tgz` file is an archive of your bundle's directory.
+    - A `{bundle_name}-{bundle_version}.tgz` file for each bundle version defined in the `yaml` file. The `.tgz` file is an archive of your bundle's directory.
 
-2. In the `values.yaml` provide your server's URL in the **repository.baseURL** attribute:
+2. In the `values.yaml` provide your server's URL in the **repository.URL** attribute:
 
   ```yaml
     repository:
-      baseURL: "http://custom.bundles-repository"
+      URL: "http://custom.bundles-repository/bundles.yaml"
   ```
+  > **NOTE:** The default filename is `index.yaml`. You can skip `yaml` file name in url if his name is `index.yaml`. In that case, do not provide the filename in the URL. Your **URL** should be equal to `http://custom.bundles-repository/`.
```

Comment:
~You can skip `yaml` file name in url if his name is `index.yaml`.~ -> I'd remove this sentence (it repeats what is said in the next one).

Reply:
For me if I remove this sentence all has no sense. The third sentence follows from the second.